cmd/tailscale/ui.go

```diff
@@ -1080,6 +1114,70 @@ func (ui *UI) layoutExitNodeDialog(gtx layout.Context, sysIns system.Insets, exi
 	})
 }
 
+// layoutPreferencesDialog lays out the preferences dialog.
+func (ui *UI) layoutPreferencesDialog(gtx layout.Context, sysIns system.Insets, exits []Peer) {
+	d := &ui.preferencesDialog
+	if d.dismiss.Dismissed(gtx) {
+		d.show = false
+	}
+	if !d.show {
+		return
+	}
+	d.dismiss.Add(gtx, argb(0x66000000))
+	layout.Inset{
+		Top:    sysIns.Top + unit.Dp(16),
+		Right:  sysIns.Right + unit.Dp(16),
+		Bottom: sysIns.Bottom + unit.Dp(16),
+		Left:   sysIns.Left + unit.Dp(16),
+	}.Layout(gtx, func(gtx C) D {
+		return layout.Center.Layout(gtx, func(gtx C) D {
+			gtx.Constraints.Min.X = gtx.Dp(unit.Dp(250))
```

Comment:
Could this be made wide enough that "Allow Incoming Connections" doesn't wrap?

![Screenshot_20230402-163534](https://user-images.githubusercontent.com/1140756/229385183-5734dd97-300d-4fbb-8e93-1accfe0e03c5.png)

Reply:
I am looking into it.

Reply:
I have made the box bigger. Sadly, I cannot test in your case, since on my Pixel 3XL it looked fine even before the fix. The 250dp measurement was used because it is used in multiple places, in the app. In the future, we should probably make the box a fraction of the device width, rather than hardcoding a value.

Reply:
The text still wraps on my Pixel-3a-XL, even with 300 pixel width, but i think we'll merge it as is and try to work with it in future commits.